northstar/src/runtime/minijail.rs

```diff
@@ -273,9 +279,12 @@ impl Process {
         let stderr =
             CaptureOutput::new(OutputStream::Stderr, &manifest.name, event_tx.clone()).await?;
 
+        // Preserve the log fd to avoid missing errors between close_open_fds and execve
+        let log_fd = (*MINIJAIL_LOG_FD, *MINIJAIL_LOG_FD);
+
         let pid = jail.run_remap_env_preload(
             &init.as_path(),
-            &[(stdout.0, 1), (stderr.0, 2)],
+            &[(stdout.0, 1), (stderr.0, 2), log_fd],
```

Comment:
ah. This is the important part. :-)

Reply:
We just need the `fd` as argument for `start`.  `State` can get a `minijail: Minijail` that holds this fd initialised from `state::State::init`

```rust
structd Minijail {
  log_fd: i32,
}

impl Minijail {
  pub fn init() -> Result<Self, Error> {
   ...
  }
```